src/es/components/molecules/tab/Tab.js

```diff
@@ -0,0 +1,122 @@
+// @ts-check
+import { Shadow } from '../../web-components-toolbox/src/es/components/prototypes/Shadow.js'
+
+export default class Tab extends Shadow() {
+  constructor (options = {}, ...args) {
+    super({ importMetaUrl: import.meta.url, ...options }, ...args)
+  }
+
+  connectedCallback () {
+    if (this.shouldRenderCSS()) this.renderCSS()
+
+    const selectors = {
+      button: 'button',
+      buttonActive: 'button.active',
+      contentActive: 'div[tab-content-target]#'
+    }
+
+    const attributes = {
+      tabTarget: 'tab-target'
+    };
+
+    const states = {
+      active: 'active',
+      show: 'show'
+    };
+
+    // Show content of default active tab
+    this.tabs = this.root.querySelectorAll(selectors.button);
+    this.tabActive = this.root.querySelector(selectors.buttonActive);
+    this.tabActiveId = this.tabActive.getAttribute(attributes.tabTarget);
+    this.contentActive = this.root.querySelector(`${selectors.contentActive}${this.tabActiveId}`);
+    this.contentActive.classList.add(states.show);
+
+    // Handle changing active tabs
+    this.tabs.forEach(tab => {
+      tab.addEventListener('click', () => {
+        this.tabActive.classList.remove(states.active);
+        this.contentActive.classList.remove(states.show);
+
+        tab.classList.add(states.active);
+        this.tabActive = tab;
+        this.tabActiveId = tab.getAttribute(attributes.tabTarget);
+        this.contentActive = this.root.querySelector(`${selectors.contentActive}${this.tabActiveId}`);
+        this.contentActive.classList.add(states.show);
+      });
+    });
+  }
+
+  disconnectedCallback () {
+    this.tabs.forEach(tab => {
+      tab.removeEventListener('click');
```

Comment:
this is not going to work. removeEventListener needs as the firgst argument the event name/type as string (ok) and as a second argument the reference to the event listener function.